processor/memorylimiter/README.md

```diff
@@ -46,6 +46,18 @@ allocated by the process heap. Note that typically the total memory usage of
 process will be about 50MiB higher than this value.
 - `spike_limit_mib` (default = 0): Maximum spike expected between the
 measurements of memory usage. The value must be less than `limit_mib`.
+- `limit_percentage` (default = 0): Maximum amount of total memory, in percents, targeted to be
+allocated by the process heap. This configuration is supported on Linux systems with cgroups
```

Comment:
Is it supported only on Linux? Or cgroups are supported only on Linux but elsewhere a percentage from the total host memory is used?

Reply:
It's supported on Linux with cgroups :). 

We could make it work on other platforms by using smth like https://github.com/pbnjay/memory. 

This feature is mostly used in dynamic environments e.g. k8s/docker.

Reply:
OK, can be added in the future. LGTM for now.